ArduCopter/GCS_Mavlink.h

```diff
@@ -23,7 +23,6 @@ class GCS_MAVLINK_Copter : public GCS_MAVLINK
 
     MAV_RESULT _handle_command_preflight_calibration(const mavlink_command_long_t &packet) override;
 
-    void send_position_target_global_int() override;
     void send_position_target_local_ned() override;
```

Comment:
Pretty sure this is a hint that your new function has to have global in it somewhere.

Perhaps we should deal with both at the same time - are the vels and accels in the same frame?

Reply:
Yes, moving local was my next step. That is why I left it the name  `get_target_info()` more generic. Dealing with the position in send_position_target_local_ned() will just need to 

Hmm on vels and accels one would hope so....?? they come into guided that way but I will check if they come out that way...